vendor/vendor.json

```diff
@@ -401,6 +401,12 @@
 			"revision": "0adb988569ffd40f68ed3422835325cfb03581c5",
 			"revisionTime": "2017-07-06T18:25:07Z"
 		},
+		{
+			"checksumSHA1": "Fas2k5UYHwTrZWoERdA4vCo6lms=",
+			"path": "github.com/taskcluster/taskcluster-worker/worker",
```

Comment:
I don't think this should sneak into dependencies...

@walac, probably you'll want to work out of a folder checked out as: `$GOPATH/src/github.com/taskcluster/taskcluster-worker/` maybe even just use this repo as the upstream repo..

I generally, don't fork repos to my own user if I have commit access to them.. for golang it gets really dirty if you check things out under a different repo name...